edc-client/src/main/java/com/turn/edc/storage/StorageConnector.java

```diff
@@ -44,4 +46,9 @@ public void handleStoreRequest(StoreRequest request) {
 			LOG.debug(ExceptionUtils.getStackTrace(e));
 		}
 	}
+
+	public void setTTL(String key, int timeout) throws KeyNotFoundException, TimeoutException, IOException {
```

Comment:
I think this is the function with "TO-DO" that I mentioned last time. I will remove this!